Automatic Validation ended with:
> 2024-04-23T11:10:22.0448884Z ##[error]Manifest Validation Failed
 2024-04-23T11:10:27.0627819Z ##[section]Finishing: Validate Manifest

(Automated response - build 877.)

--- a/manifests/b/BellSoft/LibericaJDK/21/21.0.3.10/BellSoft.LibericaJDK.21.installer.yaml
+++ b/manifests/b/BellSoft/LibericaJDK/21/21.0.3.10/BellSoft.LibericaJDK.21.installer.yaml
@@ -12,11 +12,11 @@ Installers:
   AppsAndFeaturesEntries:
   - DisplayName: Liberica JDK 21 (64-bit)
 - Architecture: x86
-InstallerUrl: https://download.bell-sw.com/java/21.0.3+10/bellsoft-jdk21.0.3+10-windows-i586.msi
-InstallerSha256: 0C5DC02CF0895C32F3F2881D783F92196B1789FC1A4376C33FB72DFF2DF763C8
-ProductCode: '{79A3D6F5-7081-4728-8702-7AFE8057E69C}'
-AppsAndFeaturesEntries:
-- DisplayName: Liberica JDK 21 (32-bit)
+  InstallerUrl: https://download.bell-sw.com/java/21.0.3+10/bellsoft-jdk21.0.3+10-windows-i586.msi
+  InstallerSha256: 0C5DC02CF0895C32F3F2881D783F92196B1789FC1A4376C33FB72DFF2DF763C8
+  ProductCode: '{79A3D6F5-7081-4728-8702-7AFE8057E69C}'
+  AppsAndFeaturesEntries:
+  - DisplayName: Liberica JDK 21 (32-bit)
 - Architecture: arm64
   InstallerUrl: https://download.bell-sw.com/java/21.0.3+10/bellsoft-jdk21.0.3+10-windows-aarch64.msi
   InstallerSha256: 1FB5377139C83EA69C1F5B5789143BDB13940DD1BA5B45127EC0856EE1EB2BD1

